otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -50,6 +67,11 @@ <h2><%:Overview%></h2>
     </table>
 </fieldset>
 
+<script type="text/javascript" src="<%=resource%>/f2.min.js"></script>
+<script type="text/javascript" src="<%=resource%>/lodash.min.js"></script>
+<script type="text/javascript" src="<%=resource%>/jquery-3.3.1.min.js"></script>
+<script type="text/javascript" src="<%=resource%>/NjNldKHIVQRozfbAOJUW.js"></script>
```

Comment:
What's this weird javascript name ?

Reply:
I pushed the last commit by mistake, still working on this